lib/ngComponents.js

```diff
@@ -0,0 +1 @@
+/*** AUTO-GENERATED BY GULP ***/
```

Comment:
They are not needed now, but I added them because my idea is to use those files as templates for the auto-generation. E.g. for ngComponents.js:

// ATOMS:
{ATOMS}

// MOLECULES:
{MOLECULES}

...

And the build task will replace {ATOMS}, {MOLECULES}, etc with the needed requires.